urdf2webots/importer.py

```diff
@@ -47,16 +54,59 @@ def mkdirSafe(directory):
             print('Directory "' + directory + '" already exists!')
 
 
-def convert2urdf(inFile, outFile=None, robotName=None, normal=False, boxCollision=False,
+def convertUrdfFile(input = None, output=None, robotName=None, normal=False, boxCollision=False,
                  disableMeshOptimization=False, enableMultiFile=False,
                  toolSlot=None, initTranslation='0 0 0', initRotation='0 0 1 0',
                  initPos=None, linkToDef=False, jointToDef=False):
-    if not inFile:
-        sys.exit('Input file not specified (should be specified with the "--input" argument).')
-    if not os.path.isfile(inFile):
-        sys.exit('Input file "%s" does not exists.' % inFile)
-    if not inFile.endswith('.urdf'):
-        sys.exit('"%s" is not an URDF file.' % inFile)
+    """Convert a URDF file into a Webots PROTO file or Robot node string."""
+    urdfContent = None
+    if not input:
+        print('''"--input" not specified, a URDF content will be read in the in stdin.\n
+            The "</robot>" tag will stop the lecture.''')
```

Comment:
```suggestion
            The "</robot>" tag will stop the reading.''')
```
Also, are you sure the newline inside the string is OK, I am afraid it won't display well, e.g., add extra spaces...